include/openssl/conf_api.h

```diff
@@ -7,40 +7,4 @@
  * https://www.openssl.org/source/license.html
  */
 
-#ifndef  OPENSSL_CONF_API_H
-# define OPENSSL_CONF_API_H
-# pragma once
-
-# include <openssl/macros.h>
-# ifndef OPENSSL_NO_DEPRECATED_3_0
-#  define HEADER_CONF_API_H
-# endif
-
-# include <openssl/lhash.h>
-# include <openssl/conf.h>
-
-#ifdef  __cplusplus
-extern "C" {
-#endif
-
-/* Up until OpenSSL 0.9.5a, this was new_section */
-CONF_VALUE *_CONF_new_section(CONF *conf, const char *section);
-/* Up until OpenSSL 0.9.5a, this was get_section */
-CONF_VALUE *_CONF_get_section(const CONF *conf, const char *section);
-/* Up until OpenSSL 0.9.5a, this was CONF_get_section */
-STACK_OF(CONF_VALUE) *_CONF_get_section_values(const CONF *conf,
-                                               const char *section);
-
-int _CONF_add_string(CONF *conf, CONF_VALUE *section, CONF_VALUE *value);
-char *_CONF_get_string(const CONF *conf, const char *section,
-                       const char *name);
-long _CONF_get_number(const CONF *conf, const char *section,
-                      const char *name);
-
-int _CONF_new_data(CONF *conf);
-void _CONF_free_data(CONF *conf);
-
-#ifdef  __cplusplus
-}
-#endif
-#endif
```

Comment:
Something to be noted is that you have not *deprecated* the `_CONF` functions, you have *removed* them from public view.  Considering how so many seem to be quite confused regarding deprecation vs removal, I think you need to be more precise about what this is doing, in commit message in and CHANGES.

Reply:
rewrote the commit message, thanks.